inf-terraform-aws/files/AUTHORS

```diff
@@ -0,0 +1 @@
+Frank Joas <frank.joas@boehringer-ingelheim.com>
```

Comment:
Not sure if this makes sense here, but it would be good to add this to the `OWNERS` filer

Reply:
https://github.com/nichtraunzer/ods-quickstarters/commit/0b904aba383c72ec5ab584146288a5cc85e943a1